modules/libcom/src/cxxTemplates/tsSLList.h

```diff
@@ -311,13 +311,13 @@ inline bool tsSLIterConst<T>::valid () const
 template < class T >
 inline bool tsSLIterConst<T>::operator == ( const tsSLIterConst<T> &rhs ) const
 {
-    return this->pEntry == rhs.pConstEntry;
+    return this->pEntry == rhs.pEntry;
 }
 
 template < class T >
 inline bool tsSLIterConst<T>::operator != (const tsSLIterConst<T> &rhs) const
 {
-    return this->pEntry != rhs.pConstEntry;
+    return this->pEntry != rhs.pEntry;
```

Comment:
Well, this fixes a latent compile error introduced by b67619c0fc2475ae6d9156771c77ae4c2b2559f7, almost 24 years ago.

Can you say how you noticed this?  Did you find a compilation error?

It should be reiterated that usage of `cxxTemplated/` is not recommended for new code.

Reply:
I just compiled with clang 19.1.5. Not sure why it decided to start evaluating this code though.

Reply:
I won't insist on a separate PR for this straightforward change.  Still, could you quote the specific error, which I assume will some variation on "no such member pConstEntry", and attach the full error output?  This will help google to help others to find this issue.

Reply:
```
clang++  -D_GNU_SOURCE -D_DEFAULT_SOURCE          -D_X86_64_ -DUNIX  -Dlinux   -DBUILDING_libCom_API   -O3 -g   -Wall      -mtune=generic     -m64  -fPIC -I. -I../O.Common -I. -I../osi/compiler/clang -I../osi/compiler/default -I. -I../osi/os/Linux -I../osi/os/posix -I../osi/os/default -I.. -I../as -I../bucketLib -I../calc -I../cvtFast -I../cppStd -I../cxxTemplates -I../dbmf -I../ellLib -I../env -I../error -I../fdmgr -I../flex -I../freeList -I../gpHash -I../iocsh -I../log -I../macLib -I../misc -I../osi -I../pool -I../ring -I../taskwd -I../timer -I../yacc -I../yacc -I../yajl -I../../../../include/compiler/clang -I../../../../include/os/Linux -I../../../../include         -c ../cxxTemplates/resourceLib.cpp
In file included from ../cxxTemplates/resourceLib.cpp:15:
In file included from ../cxxTemplates/resourceLib.h:43:
../cxxTemplates/tsSLList.h:314:32: error: no member named 'pConstEntry' in 'tsSLIterConst<T>'
  314 |     return this->pEntry == rhs.pConstEntry;
      |                            ~~~ ^
../cxxTemplates/tsSLList.h:320:32: error: no member named 'pConstEntry' in 'tsSLIterConst<T>'
  320 |     return this->pEntry != rhs.pConstEntry;
      |                            ~~~ ^
2 errors generated.
```

Reply:
Actually, I've pulled this commit into #579, just so there's less going on in this PR. Sorry for the spam.